src/Features/Core/Portable/CodeRefactorings/MoveType/AbstractMoveTypeService.MoveTypeEditor.cs

```diff
@@ -234,17 +234,28 @@ private ISet<SyntaxNode> GetMembersToRemove(SyntaxNode root)
                 return removableCandidates;
             }
 
-            private static bool FilterToTopLevelMembers(SyntaxNode node, SyntaxNode typeNode)
+            private static bool IsRemovable(SyntaxNode node, SyntaxNode typeNode, ISyntaxFactsService syntaxFacts)
             {
                 // We never filter out the actual node we're trying to keep around.
                 if (node == typeNode)
                 {
                     return false;
                 }
 
-                return node is TTypeDeclarationSyntax ||
-                       node is TMemberDeclarationSyntax ||
-                       node is TNamespaceDeclarationSyntax;
+                // IsTypeDeclaration handles ClassBlock but not ClassStatement for VB.
```

Comment:
`TNamespaceDeclarationSyntax` is no longer used after this change?

Reply:
Yeah it seems this is the case with `TNamespaceDeclarationSyntax`, `TMemberDeclarationSyntax`, and  `TCompilationUnitSyntax`. Will remove them when looking in the current legitimate test failures 😢

Reply:
@davidwengier It's used in:

https://github.com/dotnet/roslyn/blob/6da1274c9d24c2f90a48290394a951b23617f2a3/src/Features/Core/Portable/CodeRefactorings/MoveType/AbstractMoveTypeService.MoveTypeNamespaceScopeEditor.cs#L44-L45